src/components/account/list/UserRow.vue

```diff
@@ -78,7 +81,9 @@
             const isAdmin = props.user.role === Role.ADMIN;
             const student = props.user as Student;
 
-            return { editAccount, isStudent, isLecturer, isAdmin, student };
+            profilePicture.value = process.env.VUE_APP_API_BASE_URL + "/user-management/users/" + props.user.username + "/thumbnail?";
```

Comment:
```suggestion
            profilePicture.value = process.env.VUE_APP_API_BASE_URL + "/user-management/users/" + props.user.username + "/thumbnail?" + new Date().getTime();
```
This way, the thumbnail is updated in the account list. Otherwise, the browser will use the cached picture as the src url does not change

Reply:
will be handled by https://github.com/upb-uc4/lagom-core/issues/355